### DISCUSSION TITLE
Suggestion for Monthly Income-Based SFL Buyback and Holding

### DISCUSSION THREAD

**Discussion by Jericho-rpc** [sunflower-land/sunflower-land#4302] [Polls]
As the Sunflower Land project continues to grow, and with the expansion of the community, we believe it is essential to further strengthen the long-term value and stability of the SFL token. By using a portion of the project’s monthly income to buy back and hold SFL, the project team can bring about positive impacts in several areas.

We suggest that the Sunflower Land project team allocate [suggested percentage, e.g., 10%] of the monthly income to buy back SFL tokens and hold them in an account controlled by the project team. This measure will:

Enhance Token Value: Reducing the circulating supply through buybacks will increase the scarcity and potential value of SFL.
Stabilize the Market: Regular buyback activities will provide consistent demand in the market, helping to mitigate SFL price volatility.
Boost Community Confidence: The project team holding SFL tokens demonstrates confidence in the long-term development of the project, which will help build trust and support within the community.

At the beginning of each month, the project team will determine the amount of SFL to buy back based on the total income from the previous month.
The buyback will be conducted on [specific date, e.g., the 5th of each month], with the relevant data made public.
After the buyback, the SFL tokens will be held in an account controlled by the project team, with regular reports on the holdings shared with the community.

We encourage community members to actively participate in the discussion and voting on this proposal. If you support this proposal, please cast your vote in favor. We believe this measure will contribute to the sustainable development of the Sunflower Land project and provide greater long-term value for all SFL holders.

**Comment by brian-man** [comment]
Yuck.  Please no.

**Comment by Amber2234** [comment]
What’s the point of this? You just take SFL off the table and fire up a pump and dump model, punishing those who provided liquidity first (impermanent loss). Yeah, it’s pushing SFL to the moon and rewarding the holders of SFL coins. But it’s no reward for the players from whom most of the money comes.

That said, I find it perfectly okay for the team to buy back SFL for specific reasons. For example, to gift it to team members to allow them to share in the success of the game. But to have a measurable effect, the market cap should first be in a range that supports this. Additionally, you’re tempting many loyal holders of SFL to sell their coins, where the advantage isn’t clear to me.

The best option is if the SFL coin develops independently, purely from the supply and demand within the game. And that seems to be working perfectly right now.

**Reply by Jericho-rpc** [reply]
Because game tokens can be added at will, how do you avoid developers spamming tokens? I don't think it's fair enough to sfl holders, because token additions can be changed at any time, how to restrain the developers?

**Comment by bichobravo** [comment]
Why would a player want the token to go up in price? The best players want to play the game and know that everything they have created and achieved has some real value. The economy just has to work and this solution offers no benefit to the game itself.

**Reply by Jericho-rpc** [reply]
at will and issue all sorts of inflationary nft. I don't think there is any problem with the company holding additional tokens.
1. players don't want tokens to be issued indiscriminately leading to inflation
2. Players create a lot of profit every month.
3. Increase confidence in holding tokens
